### DISCUSSION TITLE
[Defibulator Bugs]

### DISCUSSION THREAD

**Discussion by GhostsModels** [EtherealEmil/UsualScrap#43] [Ideas]
A few bugs I have discovered upon playing with the defibulator

**Comment by GhostsModels** [comment]
Sometimes when reviving a player, a clone of their body is left behind. This body clone is also what is teleported when the player is selected

**Reply by EtherealEmil** [reply]
I will keep an eye on this and try to get it fixed.

**Comment by GhostsModels** [comment]
The defibulator revives players even when not correctly charged up. In fact when revived this way, players aren't put in critical condition

**Reply by EtherealEmil** [reply]
Did this happen with only this mod and its dependencies?

**Reply by GhostsModels** [reply]
I had other mods, but none that should affect reviving, so I'm not sure. I'd have to test a lot more. It wasn't consistent and it might have been related to more company or something

**Reply by EtherealEmil** [reply]
This has been fixed.
